src/test/java/uk/nhs/prm/deduction/e2e/ehr_out_db/Acknowledgement.java

```diff
@@ -0,0 +1,29 @@
+package uk.nhs.prm.deduction.e2e.ehr_out_db;
+
+import lombok.AllArgsConstructor;
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+import javax.persistence.Entity;
+import javax.persistence.Id;
+import javax.persistence.Table;
+import java.time.LocalDateTime;
+import java.util.UUID;
+
+
+@Entity
+@Getter
+@NoArgsConstructor
+@AllArgsConstructor
+@Table(name = "acknowledgements")
+public class Acknowledgement {
+    @Id
+    private UUID messageId;
```

Comment:
If we keep this as a UUID datatype, I'd add the following annotation.

```suggestion
    @Id
    @Type(type="pg-uuid")
    private UUID messageId;
```